test/functional/test_framework/test_framework.py

```diff
@@ -104,6 +104,7 @@ def __init__(self):
         """Sets test framework defaults. Do not override this method. Instead, override the set_test_params() method"""
         self.chain = 'regtest'
         self.setup_clean_chain = False
+        self.mock_time_on_setup = True
```

Comment:
@UdjinM6 could you please suggest if the proposed option is the right way to go here?
Before this change, Dash functional tests would always mock time, whereas original Bitcoin tests do it case-by-case.

Reply:
That's might be an interesting approach but it should be implemented in a separate PR imo (and applied to other tests that override `setup_network` to use `disable_mocktime`). For this PR I would rather go with https://github.com/UdjinM6/dash/commit/fa4797b5bab60b89edc4a9943ee2bc4e9635153e